database/pom.xml

```diff
@@ -44,8 +44,8 @@
         <version>${version.h2}</version>
       </dependency>
       <dependency>
-        <groupId>com.oracle.ojdbc</groupId>
-        <artifactId>ojdbc8</artifactId>
+        <groupId>com.oracle.database.jdbc</groupId>
+        <artifactId>ojdbc11</artifactId>
```

Comment:
❓ I can't find the library in our Artifactory. Do you know if that's on purpose?

Reply:
How will a new Maven dependency usually be added to Artifactory? Isn't this something that happens automatically? Right now, it's pulled from Maven Central. Maybe we need to merge with the master before the library is added to Artifactory?